QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,91 @@
+
+
+This document is a declaration of software quality for the `libconsole-bridge-dev` ROS external dependency, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `libconsole-bridge-dev` Quality Declaration
+
+The ROS external dependency `libconsole-bridge-dev` **does not claim** to be in the **Quality Level 1** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [External dependencies requirements for Quality Level 1 in REP-2XXX](https://www.ros.org/reps/rep-2004.html).
+
+External dependencies are evaluated on a case by case basis, some items may not adhere to standards defined for ROS packages.
+
+## Version Policy
+
+### Version Scheme
+
+It is not publicly available the information regarding `libconsole-bridge-dev` uses a versioning system to manage automatically their version schemes. Their latest library release is set to 0.4.4
+
+### API Stability // ABI Stability
+
+`libconsole-bridge-dev` library do not make any declaration regarding if it's API or ABI stable.
+
+### Public API Declaration
+
+There is no specific documentation regarding a Public API or expected functions that can be used from the library. 
+
+`console.h` is available in the source code, the functions available can be seen there, but the API is not documented.
+
+## Change Control Process
+
+There is no specific implementation/guidelines stated as to how authors are maintaining that library.
+
+## Documentation
+
+`libconsole-bridge-dev` does not provide any type of documentation of how the library is used in their source code repository. There is a [Wiki](http://wiki.ros.org/console_bridge) related to their functionality and expected behavior for its old ROS package.
+
+Feature and API documentation are not addressed.
+
+### License and Copyright Statements
+
+It is not explicitly stated a License or Copyright statement for the library. There is no particular LICENSE file in the source files or on the repository of the library. However, the source files (.cpp and .hpp) all contain the following BSD license header: 
```

Comment:
Actually for this library shouldn't be a problem to deal with the copyright issues. But it would be great to define the minimum requirements for a library to declare the license used.